Makefile

```
@@ -13,7 +13,11 @@ bash:
 
 .PHONY: ci
 ci:
-	@$(BASH_TEST) "bin/rails db:setup db:migrate ci"
+	@$(BASH_TEST) "bin/rails db:setup db:migrate assets:precompile ci"
+
+.PHONY: ci_local
+ci_local:
+	@$(BASH_TEST) "RAILS_ENV=test CI=true bin/rails db:create db:schema:load db:migrate assets:precompile ci"
```

I'm not a Make expert, but it looks like this works since the only thing different is the `CI` env variable:
```suggestion
	$(MAKE) ci CI=true"
```

---

changed